cmd/tink-server/main.go

```diff
@@ -68,11 +65,10 @@ func (c *DaemonConfig) PopulateFromLegacyEnvVar() {
 	c.PGSSLMode = env.Get("PGSSLMODE", c.PGSSLMode)
 	c.OnlyMigration = env.Bool("ONLY_MIGRATION", c.OnlyMigration)
 
-	c.TLSCert = env.Get("TINKERBELL_TLS_CERT", c.TLSCert)
 	c.CertDir = env.Get("TINKERBELL_CERTS_DIR", c.CertDir)
 	c.GRPCAuthority = env.Get("TINKERBELL_GRPC_AUTHORITY", c.GRPCAuthority)
 	c.HTTPAuthority = env.Get("TINKERBELL_HTTP_AUTHORITY", c.HTTPAuthority)
-	c.TLS = env.Bool("TINKERBELL_LS", c.TLS)
+	c.TLS = env.Bool("TINKERBELL_TLS", c.TLS)
```

Comment:
This maybe strengthens the argument for a `const` here

Reply:
actually not really, at least not without a bunch of weird changes to the package structure. "TINKERBELL_TLS" only occurs once in each package so the `const` wouldn't really be of much actual use, I could have likely typo'd it when constifying it :D. I think I'll go ahead with the constification of all the env strings anyway as that would at least more likely to be copy/pasted and avoid typos in that way.

Reply:
I changed my mind on this. The gain was super minimal because each string is only ever used once anyway and the typo can just happen when const'ing vs how its done now. It also was weird because at first I was going to go with names matching the strings (`const TINKERBELL_TLS = "TINKERBELL_TLS"` for e.g.). That would mean the strings would be exported and I didn't really want to do that, so I prefixed the names with `e` for env, but then that sort of looks like an error not env and it just looked weird when using it (`c.TLS = env.Bool(eTINKERBELL_TLS, c.TLS)` ) anyway. So I decided meh nm just leave it.

But now that I wrote all of that and I thought about "is going to be exported" I think I actually like that. It would force adding documentation to the exported consts which would serve as documentation for godoc readers vs `--help/-h` output. So I will do this, but tomorrow.